samples/terminal/src/main/java/com/kody/ExampleRequestRefund.java

```diff
@@ -3,66 +3,50 @@
 import com.kodypay.grpc.pay.v1.RefundResponse;
-import io.grpc.ManagedChannel;
 import io.grpc.ManagedChannelBuilder;
 import io.grpc.Metadata;
 import io.grpc.stub.MetadataUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-import java.util.Properties;
 import java.util.concurrent.TimeUnit;
 
 public class ExampleRequestRefund {
-    private static final Logger LOG = LoggerFactory.getLogger(ExampleRequestRefund.class);
-    private static final long TIMEOUT_MS = java.time.Duration.ofMinutes(3).toMillis();
+    //TODO: Replace this with the testing or live environment
+    public static final String HOSTNAME = "grpc-staging.kodypay.com";
+    public static final String API_KEY = "API KEY";
 
     public static void main(String[] args) {
+        //TODO: Replace this with your Store ID
+        String storeId = "STORE ID";
+        //TODO: Replace this with your Order ID
+        String orderId = "ORDER ID";
+        //TODO: Replace this with your amount
+        String amount = "1000";
+
+        requestRefund(storeId, orderId, amount);
+    }
 
-        Properties properties = loadProperties();
-        var address = properties.getProperty("address", "grpc-staging.kodypay.com");
-        var apiKey = properties.getProperty("apiKey");
-        if (apiKey == null) {
-            throw new IllegalArgumentException("Environment variable API-KEY is missing");
-        }
-        var storeId = properties.getProperty("storeId");
-        var orderId = properties.getProperty("orderId");
-
-        Metadata metadata = new Metadata();
-        metadata.put(Metadata.Key.of("X-API-Key", Metadata.ASCII_STRING_MARSHALLER), apiKey);
-
-        ManagedChannel channel = ManagedChannelBuilder
-                .forAddress(address, 443)
-                .idleTimeout(TIMEOUT_MS, TimeUnit.MILLISECONDS)
-                .keepAliveTimeout(TIMEOUT_MS, TimeUnit.MILLISECONDS)
-                .intercept(MetadataUtils.newAttachHeadersInterceptor(metadata))
-                .build();
-
-        KodyPayTerminalServiceGrpc.KodyPayTerminalServiceBlockingStub paymentClient = KodyPayTerminalServiceGrpc.newBlockingStub(channel);
-
-        // Define terminal request
+    private static void requestRefund(String storeId, String orderId, String amount) {
+        var paymentClient = createKodyTerminalPaymentsClient();
         RefundRequest refundRequest = RefundRequest.newBuilder()
                 .setStoreId(storeId)
-                .setAmount("1000")
+                .setAmount(amount)
                 .setOrderId(orderId)
                 .build();
-        LOG.info("requestRefund in Terminal");
-
-        RefundResponse refundRepsonse = paymentClient.refund(refundRequest).next();
-        LOG.info("refundRepsonse: {}", refundRepsonse);
 
+        System.out.println("Requesting refund...");
+        RefundResponse refundResponse = paymentClient.refund(refundRequest).next();
+        System.out.println("Refund response: " + refundResponse);
```

Comment:
change this to print out on each line each property of the response object, See the example of the PaymentDetails of the ecom